test/unittests/configuration/mycroft.yml

```diff
@@ -0,0 +1,190 @@
+Audio:
+  backends:
+    OCP:
+      active: true
+      type: ovos_common_play
+    simple:
+      active: true
+      type: ovos_audio_simple
+    vlc:
+      active: true
+      type: ovos_vlc
+  default-backend: OCP
+cache_path: /tmp/mycroft/cache
+confirm_listening: true
+date_format: MDY
+debug: false
+enclosure:
+  force_mycroft_ntp: true
+  ntp_sync_on_boot: false
+  platform: PHAL
+gui:
+  extension: generic
+  generic:
+    homescreen_supported: false
+gui_websocket:
+  base_port: 18181
+  host: 0.0.0.0
+  route: /gui
+  ssl: false
+hotwords:
+  hey mycroft:
```

Comment:
Consider putting at least one dict in `{}` to make sure it parses correctly since the spec allows either notation

Reply:
these files are the default .conf exported via the .store method, but seems unittests are not passing and we may need to change yaml lib

Reply:
Ah, this might be why I kept using `ruamel.yaml` instead of `pyyaml`.. Issue sounds familiar

Reply:
looks like `not_mycroft.yml` is exported incorrectly:
```yaml
!!python/object/new:mycroft.configuration.config.LocalConf
dictitems:
  {actual config here}
```

Reply:
Need to reproduce in tests, but this appears to be an issue with some nested dictionaries

Reply:
changed yaml lib to ruamel.yaml , seems like everything is working properly now